installers/redhat-docker/radiasoft-download.sh

```diff
@@ -126,4 +126,5 @@ EOF2
     systemctl daemon-reload
     systemctl restart docker
     systemctl enable docker
+    su vagrant
```

Comment:
The command has to be run as root. Therefore, you have to stay as root. The shells would be nested:

1. User logs in as vagrant
2. User su's as root
3. This `su vagrant`

What this should be is:

```sh
install_msg 'You will need to logout completely, and log back in.'
```

Reply:
https://github.com/radiasoft/download/pull/127/commits/ae4f9e0986d9d78a388c5fb1c27a3b252442e22e